pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
@clytwynec - do you have a recommendation here? Are there other things that fasttest does/doesn't do?